### ISSUE TITLE
feat: Zoom support?

### ISSUE THREAD

**Issue by mgenware** [cgutierr-zgz/image_compare_slider#17]
**Description**

Thanks for the great library. It would be great if we can zoom in and pan the image as well.

**Comment by lugata** [comment]
> That's a little tricky, I could investigate into it
> 
> Is wrapping it into an interactive view not enough? If not I'll give it a shot @mgenware



Yeah is not enough pls cek this dependecy in jetpack compose maybe can help u investigate
https://github.com/SmartToolFactory/Compose-BeforeAfter

**Comment by ueft-kyriv** [comment]
To add to the pan request, being able to lock the position of the left image, and then both images for panning would be a great addition. This should should add to the whole idea of side by side comparison.
